### DISCUSSION TITLE
language usage stats

### DISCUSSION THREAD

**Discussion by pchchv** [anuraghazra/github-readme-stats#2599] [Q&A] [answered]
I think I figured out what the problem was with this [issue](https://github.com/anuraghazra/github-readme-stats/issues/2583). 
If I understand correctly, language usage statistics are only collected from 100 repositories at this time. And if I understand correctly, it is not a hundred recent repositories, but a hundred first repositories.

Is it possible to increase the number of repositories? 
What do I need to do this? Do I get involved in development? Deploy on my own?

---

My Top Languages Card as an example:

[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=pchchv&theme=transparent&hide_border=true&langs_count=6&layout=compact)](https://github.com/anuraghazra/github-readme-stats)

**Now the statistics are not correct.**

**Comment by rickstaa** [comment, marked as answer]
Hey @pchchv, thanks for your feature request. This limitation is already brought to our attention and there already [exists a pull request](#2111) you can use on your own fork to get the language results for all your repositories. You can check out https://github.com/anuraghazra/github-readme-stats/issues/1852#issuecomment-1275696608 for the reason why this pull request has not yet been merged and how we are planning to solve this is the future.
